spec/features/new_project_spec.rb

```diff
@@ -49,6 +49,18 @@
     expect(File).to exist("#{project_path}/spec/support/i18n.rb")
   end
 
+  it "adds support file for view specs" do
+    run_suspenders
```

Comment:
Each `run_suspenders` takes about 30-60s, I believe? Should we use a `before` block in this spec? Could save 90+% of the test time.